### PR TITLE
Migrate community channels to markdown, update text

### PR DESCRIPTION
**Pull request by Christiandike**
Altered the structure of the documentation to improve readability and flow by making use of subheadings and concise paragraphs. Corrected wrongly placed links and converted the rst file format to markdown.

### CODE REVIEW

**Comment by Christiandike** [comment]
Hello @GeorgianaElena, please what revision do I need to make?

**Comment by GeorgianaElena** [comment]
@Christiandike, the `needs: revision` label was to signal that this PR needs review. I renamed the label to `needs: review` instead.to make it explicit. Sorry for the confusion and thanks for pointing it out.

**Comment by minrk** [comment]
This conflicts with #4065 which works on the same file, so we need to resolve that and pick one. I think it's good to switch to markdown.

**Comment by Christiandike** [comment]
> This conflicts with #4065 which works on the same file, so we need to resolve that and pick one. I think it's good to switch to markdown.

Thank you @minrk. Do i just wait? Or what next steps do I take?

**Comment by Christiandike** [comment]
> @Christiandike if you rebase and keep your community.md as-is and drop community.rst, we can merge this. I think because of the deletion, GitHub's online merge doesn't work.

Thank you @minrk ! I've successfully rebased and ready for merge

File: docs/source/contributing/community.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Community communication channels
+
+We use different channels of communication for different purposes. Whichever one you use will depend on what kind of communication you want to engage in.
+
+## Discourse
```
Comment (GeorgianaElena) [inline docs/source/contributing/community.md:5]: Let's make it even more clear that this is should be the default go-to channel

```suggestion
## Discourse (recommended)
```

File: docs/source/contributing/community.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+We use [Discourse](https://discourse.jupyter.org) for online discussions. Everyone in the Jupyter community is welcome to bring ideas and questions there.
```
Comment (GeorgianaElena) [inline docs/source/contributing/community.md:7]: I believe it is a good idea to be explicit about the fact that the Discourse channel is where support questions should live

```suggestion
We use [Discourse](https://discourse.jupyter.org) for online discussions and support questions. Everyone in the Jupyter community is welcome to bring ideas and questions there.
```

File: docs/source/contributing/community.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+Github issues are used for most long-form project discussions, bug reports and feature requests.
+
+Issues related to a specific authenticator or spawner should be opened in the appropriate repository for the authenticator or spawner. If you are using a specific JupyterHub distribution (such as [Zero to JupyterHub on Kubernetes](http://github.com/jupyterhub/zero-to-jupyterhub-k8s) or [The Littlest JupyterHub](http://github.com/jupyterhub/the-littlest-jupyterhub/)), you should open issues directly in their repository.
+
+If you cannot find a repository to open your issue in, do not worry! Open the issue in the [main JupyterHub repository](https://github.com/jupyterhub/jupyterhub/) and our community will help you figure it out.
+
+## Mailing List
+
+A [mailing list](https://groups.google.com/forum/#!forum/jupyter) for all of Project Jupyter exists, along with one for [teaching with Jupyter Notebooks](https://groups.google.com/forum/#!forum/jupyter-education).
```
Comment (GeorgianaElena) [inline docs/source/contributing/community.md:25]: Let's remove the mailing list option from this page. Min explained when the mailing list should be used in a previous comment, https://github.com/jupyterhub/jupyterhub/pull/4065#discussion_r991059181

```suggestion
```

**Review by GeorgianaElena — COMMENTED** [review]
This is looking great @Christiandike! I just left a few small suggestions.

**Comment by Christiandike** [comment]
Thank you for your suggestions @GeorgianaElena. I have applied them

**Comment by GeorgianaElena** [comment]
Thanks @Christiandike 🚀

**Pull request merged**